server/src/services/metadata.service.ts

```diff
@@ -31,19 +30,42 @@ import { JobItem, JobOf } from 'src/types';
 import { isFaceImportEnabled } from 'src/utils/misc';
 import { upsertTags } from 'src/utils/tag';
 
+const UTC_DATE_TAGS = ['GPSDateTime', 'DateTimeUTC', 'GPSDateStamp', 'SonyDateTime2'] as const;
 /** look for a date from these tags (in order) */
-const EXIF_DATE_TAGS: Array<keyof Tags> = [
-  'SubSecDateTimeOriginal',
```

Comment:
I like having the tags spelled out explicitly so we can show the users which tags are used based on the priority level